docs/index.rst

```diff
@@ -380,6 +380,19 @@ logins on the same device, you do not wish the old user to receive messages whil
 Via DRF, any creation of device with an already existing registration ID will be transformed into an update.
 If done manually, you are responsible for deleting the old device entry.
 
+MySQL compatibility
+-------------------
+MySQL has a limit for indices and therefore the `registration_id` field cannot be made unique in MySQL.
+We detect the database backend and remove the unique constraint for MySQL in the migration files. However,
+to ensure that the constraint is removed from the actual model you have to add the following to your settings:
```

Comment:
Should make it clear somehow this is only an edge case and most people even if using mysql won't need this (and mention what the use case would be)

Reply:
improved it and explained my alternative solution that even lets mysql users enjoy an indexed `registration_id`